### ISSUE TITLE
[flake] lldb-api :: tools/lldb-server/TestGdbRemote_qThreadStopInfo.py is flakey on Windows x86_64

### ISSUE THREAD

**Issue by kendalharland** [swiftlang/llvm-project#9101]

**Comment by kendalharland** [comment]
@tkremenek can you please help me by adding the `flakey-test` label to this issue, as well as the others listed in [this query](https://github.com/swiftlang/llvm-project/issues?q=is:open+is:issue+author:kendalharland+%5Bflake%5D+)? I don't have write-access to issue labels, and am working on standing up LLDB tests on our Windows x86_64 builds.

**Comment by tkremenek** [comment]
CC @adrian-prantl @JDevlieghere 

Interesting questioning what to do about tagging.  Is this a test that comes from upstream LLVM?

**Comment by adrian-prantl** [comment]
> CC @adrian-prantl @JDevlieghere
> 
> Interesting questioning what to do about tagging. Is this a test that comes from upstream LLVM?

Yes, `lldb-server` is a debugserver implementation used on all non-Darwin platforms.

@kendalharland ... which is why we probably should make that change on llvm.org and then cherry-pick it here.

**Comment by kendalharland** [comment]
@adrian-prantl this test is not flaky on Windows upstream from what I observe.  This flake only happens for me in swiftlang's llvm-project fork,, so I believe the bug belongs in this project.  For context on this issue and others: I'm currently working on getting the LLDB tests to run on the Windows x86_64 CI builds of the swift toolchain, driven by swiftlang/swift/utils/build.ps1. [See this issue](https://github.com/swiftlang/llvm-project/issues/9141).  As a baseline I'm using the results from a local llvm/llvm-project build of the LLDB tests.
